tests/admin_tests/test_actions.py

```diff
@@ -103,3 +103,28 @@ class TestRetrieveSubmissionAdminView:
 class TestRetrieveFailedSubmissionAdminView:
     # TODO: add some tests
     ...
+
+
+def test_organisationadmin_edit(setup_organisationadmin, admin_test_client):
+    view = setup_organisationadmin
+    OrganisationTypeEnum.NEW_VALUE = "New Value"
+    form_data = {
+        "organisation_name": "New Name",
+        "external_reference_code": "New Code",
+        "organisation_type": OrganisationTypeEnum.NEW_VALUE,
```

Comment:
```suggestion
        "organisation_type": OrganisationTypeEnum.NEW_VALUE.value,
```

I think this needs a `.value` at the end so that it's the actual string, rather than an Enum instance. Only the text representation would come to us over the wire, so we should send that rather than a python object.

Reply:
Changed it such that we pass the value to the form_data now, but had to create a new Enum class for this to work. With the original version, adding the value attribute it just returned an error.